src/server/scripts/Northrend/IsleOfConquest/isle_of_conquest.cpp

```diff
@@ -259,11 +259,64 @@ class spell_ioc_launch : public SpellScriptLoader
         }
 };
 
+enum SeaforiumBombSpells
+{
+    SPELL_SEAFORIUM_BLAST             = 66676,
+    SPELL_HUGE_SEAFORIUM_BLAST        = 66672,
+    SPELL_BOMB_INABLE_CREDIT          = 68366,
```

Comment:
Cosmetic detail: enums don't need more than 1 space before and after the equal sign ( = )
(unless you want to align them with any other existing nearby enums, of course).

BTW, I suggest adding `_A_` in the bomb credit name to make the names more self-evident.

```cpp
enum SeaforiumBombSpells
{
    SPELL_SEAFORIUM_BLAST       = 66676,
    SPELL_HUGE_SEAFORIUM_BLAST  = 66672,
    SPELL_A_BOMB_INABLE_CREDIT  = 68366,
    SPELL_A_BOMB_INATION_CREDIT = 68367
};
```